SharpCompress.sln

```diff
@@ -1,44 +1,44 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
Maybe it's still left over from before I tried to normalize endings but I have gitattributes set to auto.

Question is: why is this file changing at all?